### PR TITLE
feat: add internal ssh server

### PR DESCRIPTION
**Pull request by btoonk**

### CODE REVIEW

File: internal/ssh/server.go [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+			}()
+
+			wg.Wait()
+			cmd.Wait()
+
+		}
+	})
+
+	return &ssh.Server{
+		Version: "Mysocketctl-ssh-server",
```
Comment (atoonk) [inline internal/ssh/server.go:149]: I think this needs to start with SSH-2.0-
Maybe use: SSH-2.0-MySocketctl-ssh-server

File: cmd/connect.go [hunk 2]
```diff
@@ -187,7 +204,8 @@ func init() {
 	connectCmd.Flags().StringVarP(&upstream_http_hostname, "upstream_http_hostname", "", "", "Upstream http hostname")
 	connectCmd.Flags().StringVarP(&upstream_type, "upstream_type", "", "", "Upstream type: Upstream type: http, https for http sockets or mysql, postgres for database sockets")
 	connectCmd.Flags().StringVarP(&proxyHost, "proxy", "", "", "Proxy host used for connection to mysocket.io")
-	connectCmd.MarkFlagRequired("port")
+	connectCmd.Flags().BoolVarP(&localssh, "localssh", "l", false, "Proxy host used for connection to mysocket.io")
```
Comment (atoonk) [inline cmd/connect.go:207]: can we use the following as help text?
Start a local SSH server to accept SSH sessions on this host

**Review by atoonk — CHANGES_REQUESTED** [review]
looks like it may be missing the cobra flags for "mysocketctl tunnel connect"?

File: internal/ssh/server.go (hunk 1 above)
Comment (btoonk) [reply internal/ssh/server.go:149]: This ssh module will prepend this automatically.

File: cmd/connect.go (hunk 2 above)
Comment (btoonk) [reply cmd/connect.go]: nice catch, fixed.

**Review by atoonk — APPROVED** [review]

**Comment by btoonk** [comment]
@atoonk now this PR also includes the local ssh server feature in the tunnel connect command.

**Review by atoonk — APPROVED** [review]
@btoonk awesome, ship it

**Pull request merged**